clcache.py

```diff
@@ -271,8 +272,10 @@ def getManifestHash(compilerBinary, commandLine, sourceFile):
 
         commandLine.extend(collapseBasedirInCmdPath(arg) for arg in inputFiles)
 
+        printTraceStatement("Hashing commandLine: {}".format(commandLine))
         additionalData = "{}|{}|{}".format(
             compilerHash, commandLine, ManifestRepository.MANIFEST_FILE_FORMAT_VERSION)
+        printTraceStatement("Hashing additionalData: {}".format(additionalData))
```

Comment:
These two `printTraceStatement` statements seem unrelated to the commit - can you move them into a separate commit?